### DISCUSSION TITLE
TinyMCE Bundle causing javascript errors

### DISCUSSION THREAD

**Discussion by jms-gsulcer** [pimcore/pimcore#16221] [Q&A / Support] [answered]
Freshly upgraded from 10.6 to 11.1.0 and just now getting all the bundles enabled for Admin.

When I enable the TinymceBundle, I'm getting an "Uncaught Syntax Error: Unexpected end of input (at script-proxy?storage...6fb3b1625:1:1345672)

Disable the bundle and Pimcore admin loads fine, enable it and I get the script error.  I've tried removing my vendor folder and redownloading all vendor packages with a fresh composer install.

I've cleared cache and all temp files multiple times.

I can't say I understand how the scripts are loading, but from what I can tell, the bundled tinymce javascript file is where the error is.

Has anyone else seen this or is there some config setting I've missed?

**Comment by dvesh3** [comment]
@jms-gsulcer Hi Geoff, could you please try setting the `PIMCORE_DEV_MODE=1` environment var to load the unminified 
js and confirm where the error comes from?

**Reply by jms-gsulcer** [reply]
Thanks Divesh.  When I set PIMCORE_DEV_MODE=1 all scripts, including TinyMCE load without a problem. So, it seems its only the minified scripts that have an issue.

**Reply by jms-gsulcer** [reply, marked as answer]
@dvesh3 I think I tracked this down to an issue on my end. I have a response processor to remove extra whitespace and it is collapsing the minified javascript to a single line. But, the minified scripts still have single-line comments '//' so the single line minified javascript becomes unparseable.

Thanks for the help tracking this down
